### ISSUE TITLE
508-defect-2 [SEMANTIC MARKUP]: Alerts should follow design system guidance

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24637]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

There are two alerts on the "Answer psychiatry clinic questionnaire" page that do not follow design system guidance which results in a number of accessibility issues.

1. The information alert (blue note) should not use a `header` as it will add a misleading banner landmark into the middle of a screen readers landmarks list. [Read more on how to use landmarks from the W3C.](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html)

2. The warning alert should follow [warning alert guidance on the design system](https://design.va.gov/components/alertboxes#warning-alert) and include a visible warning icon for sighted colorblind users that has an `sr-only` "Alert" `span`.

<img width="1535" alt="Screen Shot 2021-05-12 at 5 42 40 PM" src="https://user-images.githubusercontent.com/14154792/118048935-04d63b80-b34b-11eb-8fde-57e45e81c57a.png">

## Acceptance Criteria

- [ ] Alerts follow design system guidance

## Steps to Recreate

Start the first of two "answer question" flows. Inspect the two alerts at the bottom of the page.

## Proposed Solution

[Please reference the design system](https://design.va.gov/components/alertboxes#warning-alert) and [use landmarks appropriately.](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html)

**Comment by mdewey** [comment]
Hey @joshkimux! This is addressed and is currently in staging!

**Comment by joshkimux** [comment]
@mdewey this is a little nitpicky, so feel free to do this last or when you have the time 🙇  

Would we be able to switch the `section` to a `div`? In this case, the alert doesn't have a heading and it doesn't start a new section on the page. That being said, the only reason the `section` is being used is for utility/styling purposes in which case it should be a `div`.

> The HTML `section` element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. **Sections should always have a heading**, with very few exceptions.
[MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

<img width="1493" alt="Screen Shot 2021-05-24 at 8 38 46 AM" src="https://user-images.githubusercontent.com/14154792/119349098-bd657e80-bc6b-11eb-9c7f-a544b0db0488.png">

**Comment by mdewey** [comment]
Feel free to nitpick away! I am fully aware I am not the best at semantic HTML and want to learn as much I can to make the review process faster and smoother :-) I will address this in my next PR since it's such a small fix.

**Comment by mdewey** [comment]
![Screen Shot 2021-05-26 at 10 40 49 AM](https://user-images.githubusercontent.com/1793923/119680046-d9555580-be0e-11eb-8afa-1800ca6cdd0f.png)

The emergency call-out is now a div :-) 

Let me know if you see anything else.

**Comment by joshkimux** [comment]
@mdewey you're on 🔥 , closing this out 🥳
